docs/node/daemon/config/settings.mdx

```diff
@@ -208,6 +237,29 @@ tls_key_file = ""
 # Kwild Server hostname
 hostname = ""
 
+#######################################################################
+###                     Extension Configuration                     ###
+#######################################################################
+[app.extensions]
```

Comment:
If we have this section, is there any standard format for an extension config to illustrate how it would look?